Docs/Book/Install.md

```diff
@@ -135,7 +127,7 @@ Due to the conda python distribution being a different version to the system pyt
 With your environment activated, this is done simply by:
 
 ```
-conda install -c rdkit rdkit-postgresql
+conda install -c conda-forge rdkit-postgresql
```

Comment:
unfortunately there's not yet a conda-forge version of the cartridge, so that needs to remain rdkit.